riscv/encoding.h

```diff
@@ -381,8 +381,12 @@
 #define MASK_ADD8 0xfe00707f
 #define MATCH_ADD_UW 0x800003b
 #define MASK_ADD_UW 0xfe00707f
+#define MATCH_ADDD 0x7b
```

Comment:
5f2a722f272ccd4ef69ec1c8c12edd1ef1648948 mentions removal of 2 P-extension instructions (pkbb32, pktt32) but I don't see that in the diff?

Reply:
The P extension evolved between the time I created the commit and the time I sent the PR. That comment was obsolete, thanks for calling it out.